drivers/interrupt_controller/exti_stm32.c

```diff
@@ -711,6 +730,35 @@ static void __stm32_exti_connect_irqs(struct device *dev)
 		CONFIG_EXTI_STM32_EXTI15_4_IRQ_PRI,
 		__stm32_exti_isr_4_15, DEVICE_GET(exti_stm32),
 		0);
+#elif defined(CONFIG_SOC_SERIES_STM32L1X)
```

Comment:
Please rebase on top of https://github.com/zephyrproject-rtos/zephyr/pull/12463 for factorization

Reply:
Okay